src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -16,10 +16,10 @@
     <NugetTargetMoniker Condition="'$(NugetTargetMoniker)'==''">.NETStandard,Version=v1.5</NugetTargetMoniker>
   </PropertyGroup>
```

Comment:
If you want the tests to actually run against netstandard1.7 you will need to add an itemgroup here that looks like:
```xml
<ItemGroup Condition="'$(TargetGroup)'=='netstandard1.7'">
    <TestNugetTargetMoniker Include="$(NugetTargetMoniker)">
      <Folder>netcoreapp1.1</Folder>
    </TestNugetTargetMoniker>
  </ItemGroup>
```

Also you will need to add the following to your project.json in the supports section:
```
"coreFx.Test.netcoreapp1.1-ns17": {
        "netstandard1.7": [
            "win7-x86",
            "win7-x64",
            "win10-arm64",
            "osx.10.10-x64",
            "centos.7-x64",
            "debian.8-x64",
            "rhel.7-x64",
            "ubuntu.14.04-x64",
            "ubuntu.16.04-x64",
            "fedora.23-x64",
            "linux-x64",
            "opensuse.13.2-x64"
        ]
    }
```